contracts/6/interfaces/IERC20EIP2612.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
```

Comment:
Remove the caret and keep the version same as in other contracts

Reply:
done, also fixed for feeProxy and other contracts